README.md

```diff
@@ -1,99 +1,79 @@
 # The TinyKV Course
 
-This is a series of projects on a key-value storage system built with the Raft consensus algorithm. These projects are inspired by the famous [MIT 6.824](http://nil.csail.mit.edu/6.824/2018/index.html) course but aim to be closer to industry implementations. The whole course is pruned from [TiKV](https://github.com/tikv/tikv) and re-written in Go. After completing this course, you will have the knowledge to implement a horizontally scalable, highly available, key-value storage service with distributed transaction support and a better understanding of TiKV implementation.
```

Comment:
We need to pinpoint the relation between TinyKV and TiKV. One of the intentions of TinyKV is to help students to understand the code of TiKV and cultivate contributors of TiKV.

Reply:
OK

Reply:
Updated.